src/components/common/CopyButton/index.tsx

```diff
@@ -1,4 +1,5 @@
-import React, { type ReactElement, ReactNode, type SyntheticEvent, useCallback, useState } from 'react'
+import type { ReactNode } from 'react'
+import React, { type ReactElement, type SyntheticEvent, useCallback, useState } from 'react'
```

Comment:
Here either.

Reply:
I won't mark the others.

Reply:
Looks like it extracts them into their own import. Can't say I like it.